wp-admin/js/widgets/media-widgets.js

```diff
@@ -397,6 +397,7 @@ wp.mediaWidgets = ( function( $ ) {
 
 				// Update widget instance.
 				control.model.set( control.getSelectFrameProps( mediaFrame ) );
+				control.model.trigger( 'update' );
```

Comment:
Why a custom `update` event here? Wouldn't it suffice to just change the above `change:attachment_id` to just `change` and then this custom event trigger could be removed?